src/nyc_trees/apps/core/templates/404.html

```diff
@@ -1,6 +1,9 @@
-{% extends "base.html" %}
+{% extends "registration/registration_base.html" %}
+{% load i18n %}
 
-{% block content %}
-    <h1>Page not found</h1>
-    <p>Sorry, but the requested page could not be found.</p>
+{% block registration_title %}{% trans "Page Not Found" %}{% endblock %}
```

Comment:
I thought translation wasn't on the table for this app, curious about the choice to use it.